packages/terra-theme-provider/tests/wdio/theme-provider-spec.js

```diff
@@ -0,0 +1,45 @@
+/* global browser, Terra, before */
+
+const viewports = Terra.viewports('tiny', 'medium');
+
+describe('Theme Provider', () => {
+  describe('Displays a default theme-provider', () => {
+    before(() => {
+      browser.url('/#/raw/tests/theme-provider/default-theme-provider');
+    });
+
+    Terra.should.matchScreenshot({ viewports });
+  });
+
+  describe('Displays a globally themed component', () => {
+    before(() => {
+      browser.url('#/raw/tests/theme-provider/global-theme-provider');
+    });
+
+    Terra.should.matchScreenshot({ viewports });
+  });
+
+  describe('Displays an updated themed component', () => {
+    before(() => {
+      browser.url('/#/raw/tests/theme-provider/switch-themes');
+    });
+
+    Terra.should.matchScreenshot({ viewports });
+  });
+
+  describe('Displays an updated globally themed component', () => {
+    before(() => {
+      browser.url('/#/raw/tests/theme-provider/global-switch-themes');
```

Comment:
We will want to switch the theme in this test to ensure the global themes were applied. Can we add a before and after screenshot of this behavior?

Reply:
60b552842fd7ee917d4d061a3de089215bca3145

Done.